src/main/webapp/app/entities/course/course-detail.component.scss

```diff
@@ -0,0 +1,15 @@
+jhi-secured-image {
+    padding-right: 20px;
+
+    ::ng-deep img {
+        border-radius: 50%;
+        height: 100px;
+        width: 100px;
```

Comment:
this is tricky in my opinion. not every icon used will be a square icon. i would just limit the height so that the image adjusts its width automatically. i would also remove the border radius

Reply:
Actually I am cropping the image to 100px width and height and keep the aspect ratio with object-fit: cover; which does not cause any problems here does it?

Reply:
No it does not cause problems. But for example the TUM logo with that border radius and in a cropped version could look quite strange

Reply:
I see, thanks for pointing out. I changed that now.